packages/test-runner-selenium/src/seleniumLauncher.ts

```diff
@@ -16,6 +16,7 @@ export class SeleniumLauncher implements BrowserLauncher {
   private iframeManager?: IFrameManager;
   private __iframeManagerPromise?: Promise<IFrameManager>;
   private isIE = false;
+  private pendingHeartbeat!: NodeJS.Timeout;
```

Comment:
It's not picking up the nodejs types, perhaps we need to configure the environment. We could also type this as `number | NodeJS.Timeout`

Reply:
Updated to use `ReturnType<typeof setInterval>` which appears to work.